pyproject.toml

```diff
@@ -71,11 +71,9 @@ mypy = "^1.9.0"
 simpleitk = "^2.3.1"
 openpyxl = "^3.1.2"
 monai = "^1.3.0"
+nnunetv2 = "^2.3.1"
 
 
-[tool.poetry.group.picai-preprocessing.dependencies]
```

Comment:
I assume a lot of the "in-house" preprocessing capabilities are replacing this guy?